### PR TITLE
fix(type): fix bluebird type

### PR DESCRIPTION
**Pull request by D-Sketon**
Promise in `document.ts` is incorrectly labeled as native Promise

### CODE REVIEW

**Review by SukkaW — APPROVED** [review]

File: src/database.ts [hunk 1]
```diff
@@ -174,11 +174,11 @@ class Database {
    * @param {function} [callback]
    * @return {Promise}
```
Comment (yoshinorin) [inline src/database.ts:175]:
```suggestion
   * @return {BluebirdPromise}
```

File: src/database.ts [hunk 2]
```diff
@@ -134,7 +134,7 @@ class Database {
...
```
Comment (yoshinorin) [inline src/database.ts:135]:
```suggestion
   * @return {BluebirdPromise}
```

Comment (SukkaW) [reply src/database.ts:135]: IMHO we can remove those JSDoc altogether and only uses TypeScript types.

**Comment by D-Sketon** [comment]
Test failed.
> Not found in manifest. Falling back to download directly from Node
> Error: Unable to find Node version '14' for platform darwin and architecture arm64.

**Comment by yoshinorin** [comment]
> Test failed.

IMHO, it's okay to merge because it's caused by CI env. Probably same issues occur in other repositories, so I think it's okay to start to drop Node.js 14 and 16.

**Review by SukkaW — APPROVED** [review]

**Pull request merged**